tests/ui/test_pvc_ui.py

```diff
@@ -196,3 +196,86 @@ def test_create_resize_delete_pvc(
         pvcs = [pvc_obj for pvc_obj in pvc_objs if pvc_obj.name == pvc_name]
         if len(pvcs) > 0:
             assert f"PVC {pvcs[0].name} does not deleted"
+
+    @tier1
+    @pytest.mark.parametrize(
+        argnames=["sc_name", "access_mode", "clone_access_mode"],
+        argvalues=[
+            pytest.param(
+                "ocs-storagecluster-ceph-rbd",
+                constants.ACCESS_MODE_RWO,
+                constants.ACCESS_MODE_RWO,
+            ),
+            pytest.param(
+                "ocs-storagecluster-cephfs",
+                constants.ACCESS_MODE_RWX,
+                constants.ACCESS_MODE_RWO,
+            ),
+        ],
+    )
+    def test_clone_pvc(
+        self,
+        project_factory,
+        teardown_factory,
+        setup_ui,
+        sc_name,
+        access_mode,
+        clone_access_mode,
+    ):
+        """
+        Test to verify PVC clone from UI
+
+        """
+        pvc_size = "1"
+        vol_mode = constants.VOLUME_MODE_FILESYSTEM
+
+        # Creating a project from CLI
+        pro_obj = project_factory()
+        project_name = pro_obj.namespace
+
+        pvc_ui_obj = PvcUI(setup_ui)
+
+        # Creating PVC from UI
+        pvc_name = create_unique_resource_name("test", "pvc")
+        pvc_ui_obj.create_pvc_ui(
+            project_name, sc_name, pvc_name, access_mode, pvc_size, vol_mode
+        )
+
+        teardown_factory(get_pvc_objs(pvc_names=[pvc_name], namespace=project_name)[0])
+
+        # Verifying PVC details in UI
+        logger.info("Verifying PVC details in UI")
+        pvc_ui_obj.verify_pvc_ui(
+            pvc_size=pvc_size,
+            access_mode=access_mode,
+            vol_mode=vol_mode,
+            sc_name=sc_name,
+            pvc_name=pvc_name,
+            project_name=project_name,
+        )
+        logger.info("Verified PVC details in UI")
+
+        # Clone PVC from UI
+        clone_pvc_name = f"{pvc_name}-clone"
+        pvc_ui_obj.pvc_clone_ui(
+            project_name=project_name,
+            pvc_name=pvc_name,
+            clone_name=clone_pvc_name,
+            clone_access_mode=clone_access_mode,
+        )
+
+        teardown_factory(
+            get_pvc_objs(pvc_names=[clone_pvc_name], namespace=project_name)[0]
+        )
+
+        # Verifying cloned PVC details in UI
+        logger.info("Verifying cloned PVC details in UI")
+        pvc_ui_obj.verify_pvc_ui(
+            pvc_size=pvc_size,
+            access_mode=clone_access_mode,
+            vol_mode=vol_mode,
+            sc_name=sc_name,
+            pvc_name=clone_pvc_name,
+            project_name=project_name,
+        )
+        logger.info("Verified cloned PVC details in UI")
```

Comment:
After cloned PVC verification, aren't we suppose to delete the PVC/Cloned PVC and the test namespace?

Reply:
@am-agrawa Yes, teardown_factory is used in line 244 and 267. Project_factory finalizer will delete the namespace.

Reply:
And what about PVC deletion? Aren't we doing that?

Reply:
Yes, teardown_factory will do that